core/dbt/events/types.proto

```diff
@@ -1585,6 +1586,85 @@ message RunResultWarningMessage {
     string msg = 2;
 }
 
+// Z047
+message DebugEnvironmentDetails {
+    EventInfo info = 1;
+    string msg = 2;
```

Comment:
All of these are msg because that's really all I'm passing to the exceptions: various messages. The messages are so flexible that trying to define messages in def message functions felt "wrong" on gut reaction. Happy to adjust!

Reply:
I think Z is fine for the debug command messages.